docs/general/ddl/data-types.rst

```diff
@@ -82,28 +82,110 @@ A basic boolean type. Accepting ``true`` and ``false`` as values. Example::
     cr> drop table my_bool_table;
     DROP OK, 1 row affected (... sec)
 
+.. _character-data-types:
+
+Character Types
+===============
+
+The general-purpose character data types available in CrateDB.
+
+Columns of below listed character data types can also be analyzed. By default
+the :ref:`plain <plain-analyzer>` analyzer is used. See
+:ref:`sql_ddl_index_fulltext`.
+
+.. _data-type-varchar:
+
+``character varying(n)``, ``varchar(n)``
+----------------------------------------
+
+The ``character varying(n)`` or ``varchar(n)`` character data types represent
+variable length strings. All unicode characters are allowed.
+
+The mandatory length specification ``n`` is an unsigned, positive `integer
+<numeric types_>`_ that defines the maximum length, in characters, of the
+values that have to be stored ot cast. The minimum length is ``1``. The
+maximum length is defined by the upper `integer <numeric types_>`_ range.
+
+An attempt to store a longer string into a column of these types will result
+in an error.
+
+::
+
+    cr> CREATE TABLE users (id varchar(2), name varchar));
+    CREATE OK, 1 row affected (... sec)
+
+::
+
+    cr> INSERT INTO users (id, name) VALUES ('1361', 'john doe')
+    SQLActionException[value too long for type character varying(2)]
+
+Unless the excess characters are all spaces, in which case the string will
```

Comment:
pg documentation says the following about this behaviour: `. (This somewhat bizarre exception is required by the SQL standard.)`